events/tracks/gta2024-ml.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: event-track
+
+title: Machine Learning
+description: his module will introduce you to the world of machine learning using Galaxy. 
+
+contributions:
+    organisers:
+        - anuprulez
+    instructors:
+        - anuprulez
+        - - hujambo-dunia
```

Comment:
```suggestion
        - hujambo-dunia
```

Reply:
hujambo-dunia also still needs to be added to the CONTRIBUTORS.yaml file

Reply:
@hujambo-dunia: Could you please add yourself to the CONTRIBUTORS.yaml file?

Reply:
@hujambo-dunia: I added you to the CONTRIBUTORS.yaml. Please check if that is ok and if you want to add more information. Thanks a lot!